pyspeckit/spectrum/writers/fits_writer.py

```diff
@@ -41,7 +41,16 @@ def write_data(self, filename=None, newsuffix='out', clobber=True,
         if self.Spectrum.xarr._make_header(tolerance=tolerance):
             for k,v in self.Spectrum.xarr.wcshead.iteritems():
                 if v is not None:
-                    header[k] = v
+                    try:
+                        header[k] = v
+                    except ValueError:
+                        try:
+                            #v is a SpectroscopicAxis
```

Comment:
that shouldn't be - header keywords have to be scalar.  It could be that `v` is a `Quantity`, though